ansible/inventory/inventory_handler.py

```diff
@@ -25,7 +25,7 @@ def fetch_tfstate(workspace: str):
 
 def get_workspace():
     cmd = "terraform workspace show"
-    working_dir = "../../terraform"
+    working_dir = "../terraform"
```

Comment:
そもそも実行ファイルからの相対パスを指定したほうが良さそう

```suggestion
    working_dir = os.path.dirname(__file__) + "/../../terraform"
```